### DISCUSSION TITLE
Как создать свой собственный список?

### DISCUSSION THREAD

**Discussion by alphagamma5** [YetiForceCompany/YetiForceCRM#15425] [How to]
Как создать свой собственный список?

**Comment by vovpff** [comment]
О чем ты?

Тут в основном общение на английском.

**Reply by alphagamma5** [reply]
уже нашла,

английский не знаю, извиняюсь,
но яндекс-браузер нормально переводит :)

**Reply by alphagamma5** [reply]
в смысле "нормально переводит" - понять на английском с их помощью могу, 
но написать с их помощью адекватно на английском пока не могу

**Reply by vovpff** [reply]
Можете на мыло писать если вопросы будут. Может что то подскажу

**Reply by alphagamma5** [reply]
Благодарю :)
